src/apps/utils/languageHelpers.js

```diff
@@ -0,0 +1,11 @@
+export function isCustomLanguage(name) {
+  return !Object.prototype.hasOwnProperty.call(CONFIG.A5E.languages, name);
+}
```

Comment:
This seems like an overly complicated approach, tbh. We could just do this:
```js
export function isCustomLanguage(name) {
  return !Object.keys(CONFIG.A5E.languages).includes(name);
}
```
However,  think `name` here is probably not the parameter name we want. Why not go for `language` since this is what these are the names of. Whether it's custom or not, it's a language.
```js
export function isCustomLanguage(language) {
  return !Object.keys(CONFIG.A5E.languages).includes(language);
}
```

Reply:
Thinking on this some more, there's actually an even better way for us to do this. Because we're working with an instance we absolutely know isn't inheriting these keys from somewhere else, it's enough to just do this:
```js
export function isCustomLanguage(language) {
  return !Object.hasOwn(CONFIG.A5E.languages, language);
}
```

Reply:
We _could_ use the `in` operator if we're willing to be a bit loose about this as well. The problem with `in` is that it's going to catch more than we want though. For example, `"constructor" in CONFIG.A5E.languages` is `true`.